examples/plotting/text.py

```diff
@@ -1,3 +1,13 @@
+''' A grid plot using descriptive text. This example demonstrates
+how to align text in a plain background in two dimensions. It also
+demonstrates how to change the angle of the texts in the plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.gridplot
+    :refs: :ref:`ug_basic_layouts_gridplot`
+    :keywords: Slider, gridplot, line, text
```

Comment:
AFAIK we have not capitalized keywords anywhere else 
```suggestion
    :keywords: slider, gridplot, line, text
```